### DISCUSSION TITLE
How is this supposed to be installed?

### DISCUSSION THREAD

**Discussion by jrodh13** [DSheirer/sdrtrunk#1927] [Q&A] [answered]
Hi there. I'm new to this program and I don't see an installer, so I'm assuming this needs to be compiled? If so, is there an install guide that would walk me though the process of compiling sdrtrunk?

**Comment by bburkett52** [comment, marked as answer]
https://github.com/DSheirer/sdrtrunk/wiki/Getting-Started

~Bob

**Comment by jrodh13** [comment]
Thank you. I didn't realize it was that simple. Although I am having trouble with getting my RSP1A working on my M1 MacBook. I'll post a new question for that.
